internal/addon/authentication/provider.go

```diff
@@ -88,24 +90,34 @@ func (sp *secretsProvider) GenerateSecrets(targetAuthType map[string]string) (ma
 		secretKeys[targetName] = secretKey
 	}
 
+	var errCount int32
 	for _, obj := range objects {
+		l := sp.log.WithValues(
+			"object_name", obj.GetName(),
+			"object_kind", obj.GetObjectKind(),
+		)
+
 		desired := obj.DeepCopyObject().(client.Object)
 		mutateFn := manifests.MutateFuncFor(obj, desired, nil)
 
 		op, err := ctrl.CreateOrUpdate(ctx, sp.k8s, obj, mutateFn)
 		if err != nil {
-			klog.Error(err, "failed to configure resource")
+			l.Error(err, "failed to configure resource")
+			errCount++
 			continue
 		}
 
-		msg := fmt.Sprintf("Resource has been %s", op)
+		msg := fmt.Sprintf("resource has been %s", op)
```

Comment:
Adding the name (maybe the namespace) of the resource can be helpful for the future

Reply:
You can see in the screenshot of the description of the PR what the logs look like. If you notice the name of the resource is already there. We could potentially add namespace but for the resources created in this function I don't think they'll be o much help

Reply:
Added namespace